docs/application/dotnet/guides/multimedia/metadata.md

```diff
@@ -156,25 +156,25 @@ To retrieve metadata from a file:
             byte[] thumbnail = metadataExtractor.GetVideoThumbnail();
             ```
 
-        - To retrieve a frame with a timestamp, use the `GetFrameAt()` method with the timestamp in milliseconds. To capture the exact frame desired, the second parameter must be set to `true`, otherwise the method returns the i-frame nearest to the desired timestamp.
+        - To retrieve a frame with a timestamp, use the `GetFrameAt()` method with the timestamp in milliseconds. To capture the exact frame desired, the second parameter must be set to `true`, otherwise the method returns the I-frame nearest to the desired timestamp:
```

Comment:
```suggestion
        - To retrieve a frame with a timestamp, use the `GetFrameAt()` method with the timestamp in milliseconds. To capture the exact frame desired, the second parameter must be set to `true`, otherwise, the method returns the I-frame nearest to the desired timestamp:
```